open-craft/opencraft
# [BB-3701] Implement Logo Update Screen page

We're fulfilling the following user story:

> "As an OCIM user, I want better-looking Logo update screen"

**References**: [BB-3701](https://tasks.opencraft.com/browse/BB-3701), [Gitlab issue](https://gitlab.com/opencraft/dev/opencraft/-/issues/702)

**Screenshots**: See Gitlab issue

**Merge deadline**: "None" 

**Testing instructions**:

1. Pull this branch to your local OCIM environment.
2. Log in with any account that has access to the console, check the Logo component on the sidebar
3. If you don't have an account set up, put a return on top of `validate_available_subdomain` in `registration/models.py`, and a dummy string in lmsUrl = props.activeInstance.data.lmsUrl; on` frontend/src/ui/components/Header/Header.tsx`. That should be enough for you to create an account.

**Author notes and concerns**:

1. This component is present on the Hero edit menu, but this was tested and works as one would expect. Styling might be a concern but that should be tackled further down the line. 

**Reviewers**
